examples/fc_poisson_analytical.py

```diff
@@ -0,0 +1,46 @@
+"""Analytical solution for Poisson process with
+   background. Produces Fig. 7 from the Feldman Cousins
+   paper."""
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy import stats
+
+from gammapy.stats import (
+    fc_find_acceptance_region_poisson,
+    fc_fix_upper_and_lower_limit,
+)
+
+fBackground  = 3.0
+fStepWidthMu = 0.005
+fMuMin       = 0
+fMuMax       = 50
+fNBinsX      = 100
+fCL          = 0.90
+
+XBins  = np.arange(0, fNBinsX)
+MuBins = np.linspace(fMuMin, fMuMax, fMuMax/fStepWidthMu + 1, endpoint=True)
+
+UpperLimitAna = []
+LowerLimitAna = []
+
+for mu in MuBins:
```

Comment:
This example took 4 minutes on my computer, without feedback to the user.
As for the other example, I'd suggest showing a progress bar and, if possible, to use a faster example.